packages/dds/tree/src/simple-tree/api/simpleSchemaToJsonSchema.ts

```diff
@@ -150,8 +154,8 @@ function convertObjectNodeSchema(schema: SimpleObjectNodeSchema): JsonObjectNode
 					};
 
 		// Don't include "description" property at all if it's not present in the input.
-		if (value.description !== undefined) {
-			output.description = value.description;
+		if (value.metadata?.description !== undefined) {
+			output.description = value.metadata.description;
```

Comment:
We're deliberately not doing anything with the rest of the metadata here, right?

Reply:
The global metadata bag isn't necessarily JSON compatible. The `description` property is explicitly respected here, but other flags are not necessarily appropriate or compatible.